api/src/main/resources/notebooks.yaml

```diff
@@ -1183,7 +1183,6 @@ definitions:
   Cluster:
     description: ''
     required:
```

Comment:
Would instead update this to reference the updated YAML per Qi's PR, these fields still exist on GetCluster